### PR TITLE
project

### PR DESCRIPTION
**Pull request by KarmaAI-soldat**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/soldat.json
Comment (CuteDog5695) [inline domains/soldat.json:1]: File is empty.

File: domains/soldat.is-a.dev
Comment (CuteDog5695) [inline domains/soldat.is-a.dev:1]: File is empty and also isn't in json format.

File: domains/subway.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Personal Project(web integration, artificial intelligence.)",
+    "repo": "https://github.com/KarmaAI-soldat/register/new/main/domains",
+    "owner": {
+        "email": "soldathayden@gmail.com"
+    },
```
Comment (CuteDog5695) [inline domains/subway.json:6]: You need to add your github username here.

File: domains/subway.json
Comment (CuteDog5695) [inline domains/subway.json:1]: I don't think we can merge this since Subway is technically trade-marked.

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/subway.json
Comment (KarmaAI-soldat) [reply domains/subway.json:1]: let me fix it

File: domains/soldat.is-a.dev
Comment (DEV-DIBSTER) [inline domains/soldat.is-a.dev:1]: Empty file.

File: domains/soldat.json
Comment (DEV-DIBSTER) [inline domains/soldat.json:1]: Empty file.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/soldat.is-a.dev
Comment (KarmaAI-soldat) [reply domains/soldat.is-a.dev:1]: how?

**Comment by omsenjalia** [comment]
Please fix your stuff and then re open the pr. It's far away from being accepted.

**Pull request closed without merging**